### PR TITLE
HW1 - Intro to MD

### PR DESCRIPTION
**Pull request by jwhenry28**

### CODE REVIEW

**Comment by jwhenry28** [comment]
This was a great assignment to learn the basics of markdown. I had trouble linking my image to the .md file, but once I obtained the correct URL it was easy.

**Comment by jwhenry28** [comment]
This was a great assignment to learn the basics of Github and Markdown. I had some trouble with figuring out the workflow, but once I familiarized myself with the process things ran smoothly.

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+  * Thomas
+  * Unnammed Guppy
+* Chickens
+  * Maple
+  * Moe
+  * Acorn
+  
+  Here is a picture of me:
+  
+  ![Headshot](https://github.com/jwhenry28/hw01-1/blob/master/Sprints%20Headshot%202019.JPG)
```
Comment (zxgneil) [inline README.md:22]: Using a relative file path may help improve easy of access of your code.

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |     x      |              |            |

Remarks:
1. Your repository is clean with no extraneous file. Good work!
1. Nice use of more than four markdown aspects of Markdown, and the introduction looks great.
1. Try using the relative file path for inserting picture may help simplify your code a bit. Give it a try next time!

**Pull request closed without merging**